parseAPI/h/CFG.h

```diff
@@ -481,6 +485,7 @@ class PARSER_EXPORT Function : public allocatable, public AnnotatableSparse, pub
     CodeObject * _obj;
     CodeRegion * _region;
     InstructionSource * _isrc;
+    bool _cache_valid;
```

Comment:
Set `_cache_valid` to true to avoid calling `finalize()` in `CudaFunction` that derived from `Function`